config/initializers/phlex.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Views
+end
+
+module Components
+  extend Phlex::Kit
+end
+
+# Rails.autoloaders.main.push_dir(
```

Comment:
it should be here?

Reply:
yessir:
<img width="1604" alt="image" src="https://github.com/user-attachments/assets/881b421e-5a04-4ac5-a1a2-d689a75b9d09">